superset-frontend/src/components/Datasource/CollectionTable.tsx

```diff
@@ -335,6 +343,13 @@ export default class CRUDCollection extends React.PureComponent<
     );
   }
 
+  getCellProps(record: any, col: any) {
+    const cellPropsFn =
+      this.props.itemCellProps && this.props.itemCellProps[col];
```

Comment:
`this.props.itemCellProps?.[col]` vs `this.props.itemCellProps && this.props.itemCellProps[col];` ?